distribution.pm

```diff
@@ -175,7 +175,7 @@ sub script_sudo {
 
 =head2 script_output
 
-    script_output($script, [ $wait, type_command => ? ]) 
+    script_output($script, [ $wait, type_command => ?, die_on_failure => ?])
```

Comment:
Update the POD explaining the parameter, and you also need to increase the [isotovideo interface](https://github.com/os-autoinst/os-autoinst/blob/master/isotovideo#L55)

Reply:
sure I can update isotovideo ver number but I thought that you updating it only in case of not backward compatible changes ?

Reply:
done

Reply:
done

Reply:
I think we increase it on every change which adds new stuff.

BTW, do we ever non-backward-compatible changes on the test API? If yes, we maybe better use [semantic versioning](https://semver.org/).

Reply:
Yes, we do backward-incompatible changes. This one is not